packages/@expo/metro-config/src/transform-plugins/import-export-plugin.ts

```diff
@@ -9,27 +9,10 @@
 // and adds support for export-namespace-from
 // https://github.com/facebook/metro/blob/8e48aa823378962beccbe37d85f1aff2c34b28b1/packages/metro-transform-plugins/src/import-export-plugin.js
 
-import type { PluginObj } from '@babel/core';
-import template from '@babel/template';
-import type { NodePath } from '@babel/traverse';
-import type {
-  ExportNamedDeclaration,
-  ImportDeclaration,
-  Program,
-  Statement,
-  SourceLocation as BabelSourceLocation,
-  Node as BabelNode,
-  Expression as BabelNodeExpression,
-  ExportAllDeclaration as BabelNodeExportAllDeclaration,
-  ExportDefaultDeclaration as BabelNodeExportDefaultDeclaration,
-  Identifier as BabelNodeIdentifier,
-  // eslint-disable-next-line import/no-duplicates
-} from '@babel/types';
-// eslint-disable-next-line import/no-duplicates
-import type * as BabelTypes from '@babel/types';
+import { template, types as t } from '@babel/core';
```

Comment:
(Discussed in DMs)
This will probably conflict with: https://github.com/expo/expo/pull/38171

The changes there aim to replace this with `type` imports. The plugin input can be changed to `ConfigAPI & typeof import('@babel/core')` after (which that PR does)

Whichever PR is merged first, will have to resolve the conflicts, but the PR linked also enforces the type-only import via a `check-packages` changes, so there's little risk of mistakingly reverting the changes from it either way 👍